pkg/ruler/ruler_test.go

```diff
@@ -431,74 +431,6 @@ func TestRuler_ExcludeAlerts(t *testing.T) {
 	}
 }
 
-func TestRuler_Rules_TenantHasEvaluationDisabled(t *testing.T) {
```

Comment:
Note that this test isn't relevant any more, because `Ruler.Rules` returns the error (this is how it used to work before my previous changes). Instead, the error is return earlier from the HTTP handler's call stack. We have unit-tests there.